chirp/drivers/radtel_t18.py

```diff
@@ -316,8 +316,9 @@ def _t18_read_block(radio, block_addr, block_size):
 def _t18_write_block(radio, block_addr, block_size):
     serial = radio.pipe
 
-    cmd = struct.pack(">cHb", 'W', block_addr, block_size)
-    data = radio.get_mmap()[block_addr:block_addr + block_size]
+    cmd = struct.pack(">cHb", b'W', block_addr, block_size)
+    data = radio.get_mmap().get_byte_compatible()[block_addr:block_addr +
```

Comment:
Okay, so ... This is technically okay, but... If you set `NEEDS_COMPAT_SERIAL=False` on your radio, then the `radio._mmap` will *always* be a MemoryMapBytes(), you can use `bytes` everywhere and not have to do this conversion here while talking to the radio.

If you're interested, setting that will get all these drivers the coveted "byte clean" designation :)